documentation/design_docs/master_variables.csv

```diff
@@ -1,16 +1,16 @@
-name,description,format,bounds,default,origin,use,current,list

```

Comment:
I wonder if we should take this opportunity to sync this master variable list with the style used in the question databases. After updating the format of the question databases for the introduction of new functions, the 'format' column here resembles the 'answer_type' column there and perhaps makes more sense to have them matching. The same for 'bounds' here and 'params' there. It is maybe more useful if these documents are consistent.